xml/obs/systemsmanagement:Agama:Staging.xml

```diff
@@ -7,10 +7,13 @@
     <flavor name="agama-live-ALP" folder="images/*/agama-live:ALP">
         <hdd filemask="agama-live.*-ALP-Build.*iso$"/>
     </flavor>
-    <flavor name="agama-live-default" folder="images/*/agama-live:default">
-        <hdd filemask="agama-live.*-default-Build.*iso$"/>
+    <flavor name="agama-live-ALP-Playwright" folder="images/*/agama-live:ALP-Playwright">
+        <hdd filemask="agama-live.*-ALP-Playwright-Build.*iso$"/>
     </flavor>
-    <flavor name="agama-live-default-Playwright" folder="images/*/agama-live:default-Playwright">
-        <hdd filemask="agama-live.*-default-Playwright-Build.*iso$"/>
+    <flavor name="agama-live-openSUSE" folder="images/*/agama-live:openSUSE">
+        <hdd filemask="agama-live.*-openSUSE-Build.*iso$"/>
+    </flavor>
+    <flavor name="agama-live-openSUSE-Playwright" folder="images/*/agama-live:openSUSEi-Playwright">
```

Comment:
There is a typo here: s/openSUSEi/openSUSE/g , let's do this by using the scripts, because manually we don't know what mistake we might do.

Reply:
Thanks, updated.